### PR TITLE
Document Streaming Runtime concepts

### PR DESCRIPTION
**Pull request by ericbottard**

### CODE REVIEW

**Review by jldec — APPROVED** [review]
awesome - thanks @ericbottard

File: docs/v0.5/runtimes/streaming.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+id: streaming
+title: Streaming Runtime
+sidebar_label: Streaming
+---
+The streaming runtime allows execution of function on **streams** of messages, permitting more complex interactions than the simple request / reply used by the [core](core.md) or [knative](knative.md) runtimes.
```
Comment (fbiville) [inline docs/v0.5/runtimes/streaming.md]: ### 
```suggestion
The streaming runtime allows execution of functions on **streams** of messages, permitting more complex interactions than the simple request / reply used by the [core](core.md) or [knative](knative.md) runtimes.
```

File: docs/v0.5/runtimes/streaming.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+deployment.extensions/franz-kafka-provisioner   1/1     1            1           23h
+
+NAME                              TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                          AGE
+service/franz-kafka-liiklus       ClusterIP      10.11.251.168   <none>           6565/TCP                         23h
+service/franz-kafka-provisioner   ClusterIP      10.11.255.151   <none>           80/TCP                           23h
+```
+
+## Streams
+
+Streams are namespaced resources that allow the flow (and typically persist) of messages serialized by riff streaming. Each stream has a **content-type** assigned to it and only messages of that MIME type are allowed on that stream.
```
Comment (fbiville) [inline docs/v0.5/runtimes/streaming.md]:
```suggestion
Streams are namespaced resources that allow the flow of (and typically persist) messages serialized by riff streaming. Each stream has a **content-type** assigned to it and only messages compatible with that MIME type are allowed on that stream.
```

File: docs/v0.5/runtimes/streaming.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+To declare a stream (and maybe provision any backing resources in the concrete message broker supporting it), use the riff cli and specify the _address of the **provisioner** service of the provider to use_.
```
Comment (fbiville) [inline docs/v0.5/runtimes/streaming.md]:
```suggestion
To declare a stream (and maybe provision any backing resources in the concrete message broker supporting it), use the riff CLI and specify the _address of the **provisioner** service of the provider to use_.
```

**Review by fbiville — APPROVED** [review]
Just minor edit suggestions

**Pull request merged**